### PR TITLE
Add new resources in Chef Infra Client 16

### PR DESCRIPTION
**Pull request by tas50**
This also fixes a few misc bugs in the old docs.

Signed-off-by: Tim Smith <tsmith@chef.io>

### CODE REVIEW

**Review by IanMadd — APPROVED** [review]

File: content/resources/cab_package/_index.md [hunk 1]
```diff
@@ -1,35 +1,38 @@
 ---
 title: cab_package resource
 resource: cab_package
-draft: false
```
Comment (phiggins) [inline content/resources/cab_package/_index.md:4]: I'm confused about this, was this intentional?

Comment (tas50) [reply content/resources/cab_package/_index.md:4]: That was probably not intentional. @IanMadd does that have a default value?

Comment (IanMadd) [reply content/resources/cab_package/_index.md:4]: It should be good. The default value is false.

Comment (tas50) [reply content/resources/cab_package/_index.md:4]: Cool we can stop generating that then. Is there a complete list of the defaults?

**Pull request merged**